decidim-elections/db/migrate/20210304121702_change_decidim_elections_trustees_name_uniqueness.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ChangeDecidimElectionsTrusteesNameUniqueness < ActiveRecord::Migration[5.2]
+  def change
+    add_index :decidim_elections_trustees, :name, unique: true
```

Comment:
I think we need to add the organization id as a column, otherwise this index will prevent to have trustees from different organizations and the same name.